examples/leaflet_sandwich.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Leaflet multilayer example | CartoDB.js</title>
+    <meta name="viewport" content="initial-scale=1.0, user-scalable=no" />
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
+    <link rel="shortcut icon" href="http://cartodb.com/assets/favicon.ico" />
+    <style>
+      html, body, #map {
+        height: 100%;
+        padding: 0;
+        margin: 0;
+      }
+    </style>
+
+    <link rel="stylesheet" href="http://libs.cartocdn.com/cartodb.js/v3/3.13/themes/css/cartodb.css" />
+  </head>
+  <body>
+    <div id="map"></div>
+
+    <!-- include cartodb.js library -->
+    <script src="http://libs.cartocdn.com/cartodb.js/v3/3.14.3-dev/cartodb.js"></script>
+
+    <script>
+
+      function main() {
+
+        // create leaflet map
+        var map = L.map('map', { 
+          zoomControl: false,
+          center: [43, 0],
+          zoom: 3
+        })
+
+        // add cartodb layer with multiple sublayers of different types
+        cartodb.createLayer(map, {
+          user_name: 'examples',
+          type: 'cartodb',
+          sublayers: [
+            {
+              type: "http",
+              urlTemplate: "http://{s}.basemaps.cartocdn.com/dark_nolabels/{z}/{x}/{y}.png",
+              subdomains: [ "a", "b", "c" ]
```

Comment:
These are valid attributes for http sublayers:
https://github.com/CartoDB/Windshaft/blob/master/doc/MapConfig-1.3.0.md#23-http-layers-options